### PR TITLE
Fix for #985

### PR DESCRIPTION
**Pull request by LeeKevin**
PR for #985

### CODE REVIEW

File: src/js/extensions/file-dragging.js [hunk 1]
```diff
@@ -74,17 +74,12 @@
...
-                var img = this.document.getElementById(id);
-                if (img) {
-                    img.removeAttribute('id');
-                    img.removeAttribute('class');
-                    img.src = fileReader.result;
-                }
-            }.bind(this);
+            // attach the onload event handler, makes it easier to listen in with jasmine
+            fileReader.addEventListener('load', function (e) {
+                var addImageElement = document.createElement('img');
```
Comment (nmielnik) [inline src/js/extensions/file-dragging.js]: Could you use the `this.document` reference here for creating the element?

File: spec/drag-and-drop.spec.js [hunk 2]
```diff
@@ -37,26 +37,55 @@ describe('Drag and Drop TestCase', function () {
...
+                // Pass into the event handler our dummy image source
+                eventListener.calls.mostRecent().args[1]({
+                    target: {
+                        result: 'data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7'
+                    }
+                });
+
+                // Expect that the image is inserted
                 expect(MediumEditor.util.insertHTMLCommand).toHaveBeenCalled();
             }
```
Comment (nmielnik) [inline spec/drag-and-drop.spec.js]: In the spirit of #985 could you add an assert in here to make sure `editableInput` fires whenever the image is inserted?

**Comment by nmielnik** [comment]
This is great, thanks @LeeKevin! 

Just had a couple small recommendations if you don't mind.

**Comment by nmielnik** [comment]
Looks like there's some test failure for IE9 and Edge, are you able to look into them at all?  I can try to help otherwise.

IE9
IE9

Edge
Edge

**Comment by nmielnik** [comment]
So in IE9, `FileReader` doesn't exist, so seems like the `beforeEach()` for the drag-and-drop tests should just check for the existence of `FileReader` and not try to create a spy if it doesn't exist.

The Edge issue appears to just to be a bug where our implementation of `insertHTMLCommand` doesn't trigger `editableInput`.  I'll work on a fix for that ASAP.

**Comment by nmielnik** [comment]
I believe #993 will resolve the test failure in Edge, so you'll only need to resolve the issue where `FileReader` doesn't exist in IE9 which is causing the drag-and-drop test case to fail.

**Comment by j0k3r** [comment]
@LeeKevin I guess you have to rebase your branch with our master to fix Edge test

**Comment by LeeKevin** [comment]
Hopefully this works.

**Comment by j0k3r** [comment]
The rebase looks goods.

**Comment by nmielnik** [comment]
Thanks for removing the `IE9` check, feature detection > browser sniffing.

All tests passed ([build log](https://travis-ci.org/yabwe/medium-editor/builds/111798019)) including the cross-browser testing in SauceLabs, thanks for the work @LeeKevin !

**Pull request merged**